src/main/java/peoplesoft/logic/commands/person/PersonDeleteCommand.java

```diff
@@ -42,11 +42,11 @@ public CommandResult execute(Model model) throws CommandException {
             throw new CommandException(Messages.MESSAGE_INVALID_PERSON_DISPLAYED_INDEX);
         }
 
-        Person personToDelete = lastShownList.get(targetIndex.getZeroBased());
-        model.deletePerson(personToDelete);
+        Person p = lastShownList.get(targetIndex.getZeroBased()); // p is the person to delete
```

Comment:
I don't really get the point of this change, but nbd

Reply:
to reflect what is depicted in the sequence diagram for persondelete 1